app/components/ErrorLayout/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './style.scss';
+
+const ErrorLayout = ({ error }) => {
+  if (!error) return null;
+
+  return (
+    <div className="row error-layout">
+      <img
+        src="https://minio.codingblocks.com/amoeba/404-image-min.png"
```

Comment:
Import the image in the project

Reply:
@hereisnaman, rest of the changes are made but this one for importing the image.
**Problem: **
since I can not import the  https://minio.codingblocks.com/amoeba/404-image-min.png as 
import ErrorImage from "https://minio.codingblocks.com/amoeba/404-image-min.png" goes to find out the image in ErrorLayout Component.

**Solution:**
I downloaded the image save it to the img folder where cb-logo exist and reference the image from there using

`<img src="/img/some-error.png" alt="There was some error`" />`

Is it good? if not kindly Please tell me what do you mean by import the image.

Latest code for this is :
 
![img](https://user-images.githubusercontent.com/39825643/82136215-d487ba00-9828-11ea-8781-0d063beb9611.png)

Reply:
That is exactly what you need to do. But rename the file to error-image.png

Reply:
ok thanks. Will be making a PR soon in no time. Thanks.

> @jugshaurya please do not resolve review comments, until you push the changes.

OK.Thank you 😊